serverless.yml

```diff
@@ -32,7 +32,7 @@ custom:
     includeModules: true
   # Load our secret environment variables based on the current stage.
   # Fallback to default if it is not in prod.
-  environment: ${file(env.yml):${self:custom.stage}, file(env.yml):default}
+  # environment: ${env:} #${file(env.yml):${self:custom.stage}, file(env.yml):default}
 
```

Comment:
why is it commented out?

Reply:
@n13 If you want to deploy it locally you can just uncomment this and run `sls deploy` Probably should document this